### DISCUSSION TITLE
More visually creative functions

### DISCUSSION THREAD

**Discussion by shawneeatparabol** [ParabolInc/parabol#5659] [Feedback]
## Issue - Enhancement

I have received various comments from users that they would like more styles of templates. This [customer](https://app.hubspot.com/contacts/3888472/company/3223836226) articulated it very nicely:
"It would be helpful if there were more graphics: you can set up different templates but they're all columns & text. I'm using Parabol for Lean Coffees as well as Retros but it would be nice if you could have at least a background picture you could set, something to set the mood, something to get people into a creative set-up where they're inspired... imagery triggers different emotions....In person we use post-it notes and frequently draw our ideas on them - I would love to be able to draw. We run retros every 2 weeks with the same team members and as a facilitator it would be good to have more template/style options to engage everyone and inspire them. **I want disruption**. "

### Acceptance Criteria (optional)
Users can:
- Do this: Set background pictures
- Do that: doodle on cards
- Do this: access visual templates like Sailboat

**Comment by shawneeatparabol** [comment]
This [customer](https://app.hubspot.com/contacts/3888472/contact/10972101) has also asked for the ability to be able to draw and generally have more creative options

**Comment by gcrickman** [comment]
[User](https://app.hubspot.com/contacts/3888472/contact/13282901) 🔒  feedback: 

> It might be beneficial to have a more in-depth problem solving to get to root cause of items, like a fishbone retro, we do use the comments and 5 why's to dive deeper but the fishbone allows you to visualize the problem. It would be nice to have some kind of problem-solving extension or be able to automate the 5 why's 
> 
> It would be nice to have the ability to add a picture or graphic, for our large company retro's we use some of the most known one's like sailboat, balloon, what's in the box, it would be nice to add a graphic

**Comment by gcrickman** [comment]
[User](https://app.hubspot.com/contacts/3888472/contact/11132301) 🔒  feedback:

> There's no possibility of adding pictures it would be helpful to show images.

**Comment by shawneeatparabol** [comment]
user and soon to be 🔐 [client](https://app.hubspot.com/contacts/3888472/contact/12090901) says:

I user Parabol and Miro. I prefer the dashboard, voting and general UI of Parabol over Miro but the format in Parabol is always the same. There is no surprise factor if you use just Parabol. So I go back and forth. I would like to be able to run a retro where we don't have to group everything into columns.

**Comment by gcrickman** [comment]
[A user](https://app.hubspot.com/contacts/3888472/contact/23293251/) 🔒  writes via support:

> Is it somehow possible to Post Memes in the Reflect Cards?

**Comment by gcrickman** [comment]
A [customer](https://app.hubspot.com/contacts/3888472/contact/11555401) 🔒 writes:

> Hi, what I am missing and what I would really like is to have the option to add an image to each column in a retrospective. This would personalize the retrospective even more en also make it more fun :)

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/16427351) 🔒  says: 

> We would like it if you added more templates with some graphical animations to make it fancy or funny.

[a comment by an automated account is not shown]

**Comment by acressall** [comment]
Recent users requesting "fun" things

> small animated reactions would be nice. could be too having reactions/animations incorporated while we go through the retros and click at different actions

> Change the fact that it doesn't have an integration with a gif source for all them memes!

> Prévoir un trophée pour la personne ayant les estimés les plus juste à la fin du poker planning
Translation: give a trophy to the person with the highest estimate at the end of a poker meeting

> Add a shoutout section at the end. 

> Allow more than 12 emojis on a card.

> let people vote on action items and/or emoji them

> Have the ability to play background music

**Comment by acressall** [comment]
A user writes:

> Something that would be cool is if the team could comment with gifs in retrospective cards, retro comments, or tasks

**Comment by acressall** [comment]
A user request:

> let us attach images to our comments

**Comment by acressall** [comment]
Another user request for background music

> thinking music built in

**Comment by acressall** [comment]
A user requests:

> Ability to add gifs

**Comment by jordanh** [comment]
Noting that a potential user at USAF Level Up asked for whiteboard functionality
